compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -591,6 +591,63 @@ TR::Register *OMR::X86::TreeEvaluator::fpBinaryArithmeticEvaluator(TR::Node
       }
    }
 
+TR::Register *OMR::X86::TreeEvaluator::fpNegEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   static uint8_t MASKOFFLOAT[] =
```

Comment:
Ideally, these arrays would be forced 16-byte aligned to improve the instruction that consumes them as a 16-byte quantity.  The C++11 `alignas` specifier would do the trick, but sadly not supported on all our build compilers. :-(

Reply:
Oops.  Wrong PR, but it applies to this one too.

Reply:
Agreed. Once all our build compilers have the proper support, we should update these arrays with `alignas`.